test/e2e/flask/multichain-api/wallet_createSession.spec.ts

```diff
@@ -369,8 +369,8 @@ describe('Multichain API', function () {
   });
 
   describe('Dapp has existing session with 3 scopes and 2 accounts and then calls `wallet_createSession` with different scopes and accounts', function () {
-    const OLD_SCOPES = ['eip155:1337', 'eip155:1', 'eip155:42161'];
-    const NEW_SCOPES = ['eip155:1338', 'eip155:1000'];
+    const OLD_SCOPES = ['eip155:1337', 'eip155:1338'];
```

Comment:
Updated to include only valid scopes that exist in withNetworkControllerTripleNode()
1337 (0x539), 1338 (0x53a), 1000 (0x3e8) are the chains created by the fixture